src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -121,14 +116,22 @@ ApplicationManagerImpl::ApplicationManagerImpl()
                                       &ApplicationManagerImpl::OnTimerSendTTSGlobalProperties,
                                       true),
     is_low_voltage_(false) {
+
+    devicesType.insert(
```

Comment:
@pestOO, I agree, it's good idea, but not within this task because of short estimates.